messages/gettest.md

```diff
@@ -22,7 +22,11 @@ Provide a test run ID to display test results for an enqueued or completed async
 
 - Specify a directory in which to save the test results from the org with the specified username (rather than your default org):
 
-  <%= config.bin %> <%= command.id %> --test-run-id <test run id> --code-coverage --output-dir <path to outputdir> --target-org me@myorg',
+  <%= config.bin %> <%= command.id %> --test-run-id <test run id> --code-coverage --output-dir <path to outputdir> --target-org me@myorg'
+
+- View test results. The command outputs the apex get test command with a job ID that you then run to view the full results:
```

Comment:
Hmmm, not sure why you added this example.  It doesn't show anything new (i.e. it's not using any different flags from the existing examples).  And I think the blurb is incorrect -- the command outputs the test results, not the "apex get test" command to run.  Did you mean this to be in the examples for "apex test run"?

Reply:
removed the example per our conversation.